mods/tnt/init.lua

```diff
@@ -1,10 +1,8 @@
 tnt = {}
--- Default to enabled in singleplayer and disabled in multiplayer
-local singleplayer = minetest.is_singleplayer()
-local setting = minetest.setting_getbool("enable_tnt")
-if (not singleplayer and setting ~= true) or
-		(singleplayer and setting == false) then
-	return
+-- Default to enabled when in singleplayer
+local enable_tnt = minetest.setting_getbool("enable_tnt")
+if enable_tnt == nil and minetest.is_singleplayer() == true then
```

Comment:
```minetest.is_singleplayer() == true``` is redundant, just use ```minetest.is_singleplayer()```
However if getbool returns nil in mulitiplayer then enable_tnt remains nil, t4im's suggestion works better:
```
local enable_tnt = minetest.setting_getbool("enable_tnt")
if enable_tnt == nil then
  enable_tnt = minetest.is_singleplayer()
end
```